docs/_getting-started/dev-mode.md

```diff
@@ -12,13 +12,20 @@ Starting geth in dev mode does the following:
 -   Turns off discovery by other nodes
 -   Sets the gas price to 0
 -   Uses the Clique PoA consensus engine with which allows blocks to be mined as-needed without excessive CPU and memory consumption
--   Uses on-demand block generation, producing blocks when there are transactions waiting to be mined
+-   Uses on-demand block generation, producing blocks when transactions are waiting to be mined
 
-You can specify a data directory to maintain state between runs using the `--datadir` option, otherwise databases are ephemeral and in-memory:
+## Start Geth in Dev Mode
+
+You can specify a data directory to maintain state between runs using the `--datadir` option, otherwise, databases are ephemeral and in-memory:
+
+```shell
+mkdir test-chain-dir
+```
+
+For this guide, start geth in dev mode, and enable [RPC](../_rpc/server.md) so you can connect other applications to geth. For this guide, we use Remix, the web-based Ethereum IDE, so also allow its domains to accept cross-origin requests.
 
 ```shell
-$ mkdir test-chain-dir
-$ geth --dev --datadir test-chain-dir console
+geth --datadir test-chain-dir  --port 8545 --rpc --rpcport "8545" --dev --rpccorsdomain "https://remix.ethereum.org,http://remix.ethereum.org" console
```

Comment:
`--port` is not needed here

Reply:
`--rpcport "8545"` doesnt' need the quotes, and is also not needed because that's the default port